src/main/resources/icons/io.kestra.plugin.serdes.protobuf.svg

```diff
@@ -0,0 +1,2 @@
+<?xml version="1.0" encoding="utf-8"?><!-- Uploaded to: SVG Repo, www.svgrepo.com, Generator: SVG Repo Mixer Tools -->
```

Comment:
Hmm not sure for this one is the "official" logo and if there is one actually. I'll check with @Nico-Kestra !

Reply:
done